test-case/multiple-pause-resume.sh

```diff
@@ -93,10 +93,10 @@ done
 
 # create combination list
 declare -a pipeline_combine_lst
-for i in $(sof-combinatoric.py -n ${#pipeline_idx_lst[*]} -p $max_count)
+for i in $(sof-combinatoric.py -n ${#pipeline_idx_lst[*]} -p "$max_count")
 do
     # convert combine string to combine element
-    pipeline_combine_str="$(echo $i|sed 's/,/ /g')"
+    pipeline_combine_str="${i//,/ }"
```

Comment:
Could you extend the comment and add something like "... by replacing commas with spaces for the 'for' loop below". This shell syntax is less obvious and  probably less well known that the `sed` one before.